### PR TITLE
ao

### PR DESCRIPTION
**Pull request by prhuft**
The analog output class has been finished up inasmuch as can be done without directly running tests with hardware, which I plan to wait to do until I am back in lab and can detach my AO hardware from the actual experiment (e.g. coil drivers).

### CODE REVIEW

File: analogout.py [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+                    self.startTrigger.waitForStartTrigger = self.str_to_bool(child.text)
+
+                elif child.tag == "exportStartTrigger":
+                    self.exportTrigger.exportStartTrigger = self.str_to_bool(child.text)
+
+                elif child.tag == "triggerSource":
+                    self.startTrigger.source = child.text
+
+                elif child.tag == "exportStartTriggerDestination":
+                    self.exportTrigger.outputTerminal = child.text # TODO implement the exportTrigger
```
Comment (JuanBohorquez3) [inline analogout.py]: TODO has been done?

File: analogout.py [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+                elif child.tag == "triggerEdge": # TODO: make ao_edges the correct form
+                    try:
+                        self.startTrigger.edge = StartTrigger.nidaqmx_edges[child.text]
+                    except KeyError as e:
+                        # TODO: replace with logger
+                        print(f"Not a valid {child.tag} value {child.text} \n {e}")
+                        raise
+
+                # TODO: external clock could be a class as in LabVIEW. TBD.
```
Comment (JuanBohorquez3) [inline analogout.py]: What does the labview class do? Sometimes they just store some number of variables, if that's the case, no need to make it a class. Dictionaries or rc get the job done if convenient.

**Review by JuanBohorquez3 — APPROVED** [review]
Looks good to me.

File: analogout.py (hunk 1 above)
Comment (prhuft) [reply analogout.py]: Thanks, this has been addressed now.

File: analogout.py (hunk 2 above)
Comment (prhuft) [reply analogout.py]: Yep, I used rc for this.

**Pull request merged**